### PR TITLE
[FTAMS102020] Abbie Summers

### PR DESCRIPTION
**Pull request by ANSummers**
To Do:
twoDigitsNumber()
stopClick()
resetClick()
splitClick()

### CODE REVIEW

File: javascript/chronometer.js [hunk 1]
```diff
@@ -1,15 +1,19 @@
 class Chronometer {
```
Comment (wax-bear) [inline javascript/chronometer.js:1]: Hey Abbie :),

Please name the pull request according to the following format:

[FTAMS102020] <name>

File: javascript/chronometer.js [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
   startClick(callback) {
-    // ... your code goes here
+    setInterval(
+      () => this.currentTime++, 1000)
```
Comment (wax-bear) [inline javascript/chronometer.js:8]: Great stuff!

Two matters could be improved:
1. Perhaps declare the function outside the setInterval method and submit it as the first argument of the setInterval method.
2. The callback should be (conditionally) called.

File: javascript/chronometer.js [hunk 3]
```diff
@@ -1,15 +1,19 @@
...
   }
   getMinutes() {
-    // ... your code goes here
+    let minutes = this.currentTime / 60;
+    return Math.floor(minutes);
   }
   getSeconds() {
-    // ... your code goes here
+    return this.currentTime % 60;
```
Comment (wax-bear) [inline javascript/chronometer.js:15]: Nice!

**Comment by ANSummers** [comment]
I finished Iteration 1 and made progress on the 2nd. I might need help with getting the time to display on the chronometer.

**Pull request closed without merging**